pulser-pasqal/pulser_pasqal/pasqal_cloud.py

```diff
@@ -86,14 +84,26 @@ def create_batch(
                 "must be a real device, instance of 'Device'."
             )
 
-        self._sdk_connection.create_batch(
-            serialized_sequence=seq.serialize(),
-            jobs=jobs,
+        for params in jobs:
+            seq.build(**params.parameters)  # type: ignore
```

Comment:
This passes with `runs` in the parameters (which they should always be in) because `Sequence.build()` ignores extra variables, but don't forget that `runs` is not supposed to be given to `Sequence.build()`.